### PR TITLE
Add basic telemetry submission

### PR DESCRIPTION
**Pull request by titanous**

### CODE REVIEW

File: host/cli/bootstrap.go [hunk 1]
```diff
@@ -296,6 +296,23 @@ $$;`, step.Artifact.URI, step.ID))
 		data.MongoDB.ImageArtifact.URI = artifactURIs["mongodb"]
 	}
 
+	// set TELEMETRY_CLUSTER_ID
+	telemetryClusterID := random.UUID()
+	sqlBuf.WriteString(fmt.Sprintf(`
+UPDATE releases SET env = jsonb_set(env, '{TELEMETRY_CLUSTER_ID}', '%q')
+WHERE release_id = (SELECT release_id FROM apps WHERE name = 'controller');
```
Comment (lmars) [inline host/cli/bootstrap.go]: Is the intention to explicitly change this value when restoring from a backup, or should there be a `WHERE env->>'TELEMETRY_CLUSTER_ID' IS NULL` check?

Comment (titanous) [inline host/cli/bootstrap.go]: Yes, the `TELEMETRY_CLUSTER_ID` is new for every restore, and the `TELEMETRY_BOOTSTRAP_ID` is generated once and stays the same across multiple restores.

Comment (lmars) [inline host/cli/bootstrap.go]: Ah ok, I had those two the other way round in my head.

Comment (titanous) [inline host/cli/bootstrap.go]: Yeah, they are definitely ambiguous, but this made the most sense to me.

**Pull request merged**